Duplicated dependency `commons-text` - liquibase-core

### Search first

- [X] I searched and no similar issues were found

### Description

Hello, I think `commons-text` dependecy is duplicated, its included in liquibase core and in `com.opencsv:opencsv` 

Good thing that its the same version, but in the future it can cause Jar hell.  



### Steps To Reproduce

`mvn dependency:tree`

### Expected/Desired Behavior

no duplicated deps

### Liquibase Version

4.27.0

### Database Vendor & Version

_No response_

### Liquibase Integration

_No response_

### Liquibase Extensions

_No response_

### OS and/or Infrastructure Type/Provider

_No response_

### Additional Context

_No response_

### Are you willing to submit a PR?

- [X] I'm willing to submit a PR (Thank you!)

@MalloD12 

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-enforcer-plugin:3.4.1:enforce (default-cli) on project hermesms: 
[ERROR] Rule 1: org.apache.maven.enforcer.rules.dependency.DependencyConvergence failed with message:
[ERROR] Failed while enforcing releasability.
[ERROR] 
[ERROR] Dependency convergence error for org.apache.commons:commons-text:jar:1.11.0 paths to dependency are:
[ERROR] +-pl.com.aaaaa:vvvvvv:jar:1.0.3-SNAPSHOT
[ERROR]   +-org.liquibase:liquibase-core:jar:4.27.0:compile
[ERROR]     +-com.opencsv:opencsv:jar:5.9:compile
[ERROR]       +-org.apache.commons:commons-text:jar:1.11.0:compile
[ERROR] and
[ERROR] +-pl.com.aaaaa:vvvvvv:jar:1.0.3-SNAPSHOT
[ERROR]   +-org.liquibase:liquibase-core:jar:4.27.0:compile
[ERROR]     +-org.apache.commons:commons-text:jar:1.11.0:compile
[ERROR] and
[ERROR] +-pl.com.aaaaa:vvvvvv:jar:1.0.3-SNAPSHOT
[ERROR]   +-org.apache.commons:commons-text:jar:1.12.0:compile
[ERROR] -> [Help 1]
````

--- a/liquibase-standard/pom.xml
+++ b/liquibase-standard/pom.xml
@@ -123,6 +123,14 @@
                     <groupId>commons-beanutils</groupId>
                     <artifactId>commons-beanutils</artifactId>
                 </exclusion>
+                <exclusion> <!-- direct dependency for better dependabot control -->
+                    <groupId>org.apache.commons</groupId>
+                    <artifactId>commons-text</artifactId>
+                </exclusion>
+                <exclusion> <!-- direct dependency for better dependabot control -->
+                    <groupId>org.apache.commons</groupId>
+                    <artifactId>commons-collections4</artifactId>
+                </exclusion>
             </exclusions>
         </dependency>
 
@@ -152,9 +160,24 @@
             <scope>provided</scope>
         </dependency>
         <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>${commons-lang3.version}</version>
+        </dependency>
+        <dependency>
             <groupId>commons-io</groupId>
             <artifactId>commons-io</artifactId>
             <version>${commons-io.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-collections4</artifactId>
+            <version>${commons-collections4.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-text</artifactId>
+            <version>${commons-text.version}</version>
         </dependency>
     </dependencies>
 

--- a/pom.xml
+++ b/pom.xml
@@ -71,6 +71,8 @@
         <argLine></argLine>
         <commons-lang3.version>3.14.0</commons-lang3.version>
         <commons-io.version>2.16.1</commons-io.version>
+        <commons-collections4.version>4.4</commons-collections4.version>
+        <commons-text.version>1.11.0</commons-text.version>
         <ant.version>1.10.14</ant.version>
         <junit-jupiter.version>5.10.2</junit-jupiter.version>
         <groovy.version>4.0.21</groovy.version>

